src/main/java/be/syntra/devshop/DevshopBack/controllers/ProductController.java

```diff
@@ -0,0 +1,31 @@
+package be.syntra.devshop.DevshopBack.controllers;
+
+import be.syntra.devshop.DevshopBack.models.ProductDTO;
+import be.syntra.devshop.DevshopBack.repositories.ProductRepository;
+import be.syntra.devshop.DevshopBack.services.DTOMapperService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+
+@Controller
+@RequestMapping("/products")
+public class ProductController {
+    private ProductRepository productRepository;
+    private DTOMapperService dtoMapperService;
+
+    public ProductController(ProductRepository productRepository, DTOMapperService dtoMapperService) {
+        this.productRepository = productRepository;
+        this.dtoMapperService = dtoMapperService;
+    }
```

Comment:
should we annotate this with @Autowired ? so it is clear where dependency injection takes place in out code?

Reply:
@autowire on either the private fields, or on the constructor.